Observer/OrderSaveAfter.php

```diff
@@ -57,22 +59,26 @@ function execute(Observer $ob) {
 							? '{"error": "Magento webhook failed to encode order, please look at order ' . $order_id . ' on website to see the details."}'
 							: $encoded_str;
 						$res = $this->post($payload_str, $o->getStore());
+					} catch (Throwable $e) {
+						$comment[] = 'Stock2Shop Webhook exception: ' . $e->getMessage();
 					} catch (\Exception $e) {
```

Comment:
just change Exception to Throwable

Reply:
[dd80a54](https://github.com/stock2shop/magento2_module_webhook/pull/12/commits/dd80a546ef8819e4935afcc9ead09ab404baef67)

This is the new result that is returned:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/c6c3add7-46c0-46c7-b68e-0ce23623ba30" />